src/widget/sidebar/extra.c

```diff
@@ -385,44 +394,100 @@ static int draw_request_buttons(int y_offset)
 {
     int original_offset = y_offset;
 
+    y_offset += EXTRA_INFO_VERTICAL_PADDING;
+
     for (int i = 0; i < data.visible_requests; i++) {
         const request *r = &data.requests[i];
-        y_offset += EXTRA_INFO_VERTICAL_PADDING;
+
         if (data.visible_requests < data.active_requests && i == data.visible_requests - 1) {
             text_draw_centered(translation_for(TR_SIDEBAR_EXTRA_REQUESTS_VIEW_ALL),
-                data.x_offset, y_offset + EXTRA_INFO_VERTICAL_PADDING + 4, data.width, FONT_NORMAL_GREEN, 0);
+                data.x_offset, y_offset + 25, data.width, FONT_NORMAL_GREEN, 0);
```

Comment:
The "view all" button should be vertically centered and slightly bigger.

Reply:
yes you can modify